### PR TITLE
Parameterize AMI used in AWS VPC integration tests

### PR DESCRIPTION
**Pull request by branden**
## High Level Description

This allows a client of `test_util/test_aws_vpc.py` to specify the AMI from which cluster nodes are launched.

## Related Issues

  - [DCOS_OSS-668](https://jira.mesosphere.com/browse/DCOS_OSS-668) Parameterize AMI in AWS onprem test

## Checklist for all PR's

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: This is a change to the test script API that CI will be using.
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by branden** [comment]
Here's an example build that overrides the AMI to CentOS 7.3: https://teamcity.mesosphere.io/viewLog.html?buildId=579791&buildTypeId=ClosedSource_Dcos_IntegrationTests_CloudIntegrationTests_DcosOssCcmIntegration_2&tab=buildResultsDiv

**Comment by branden** [comment]
Ready for review @mellenburg @orsenthil.

File: test_util/aws.py [hunk 1]
```diff
@@ -424,16 +424,30 @@ class VpcCfStack(CfStack):
...
+            OS_AMIS[instance_os][boto_wrapper.region],
+            instance_count,
+            admin_location,
+            key_pair_name,
+            boto_wrapper,
+        )
+        return stack, OS_SSH_INFO[instance_os]
+
+    @classmethod
+    def create_from_ami(cls, stack_name, instance_type, instance_ami, instance_count,
```
Comment (orsenthil) [inline test_util/aws.py]: [minor] - In general, it might be a good idea to write dependency (this method) above the method from it is used (`create`).

**Comment by branden** [comment]
That's an implementation detail that should hopefully be irrelevant to a client of the class. I went with this order because the `if ` statement felt a little more natural.

**Pull request merged**